docs/src/user-guide/command-line-interface.md

```diff
@@ -120,17 +130,22 @@ npx eslint --ext .jsx,.js lib/
 
 Disables use of configuration from `.eslintrc.*` and `package.json` files.
 
-Example:
+* **Argument Type**: No argument.
+
+##### `--no-eslintrc` example
 
 ```shell
 npx eslint --no-eslintrc file.js
 ```
 
-#### `-c`, `--config`
+#### `--config`, `-c`
```

Comment:
Perhaps we should keep `` `-c`, `--config` `` (and others) in that order to match the output of `eslint -h` where short flags come first?

Reply:
Agree

Reply:
ok, switching order